GameOfThrones/Views/Castle/Index.cshtml

```diff
@@ -0,0 +1,17 @@
+﻿@model IEnumerable<GameOfThrones.Models.Castle>
+
+@{
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+@{ 
+    ViewBag.Title = "Castles";
+}
```

Comment:
These could be combined into a single block
```suggestion
@{
    ViewBag.Title = "Castles";
    Layout = "~/Views/Shared/_Layout.cshtml";
}
```

Reply:
Combined into a single block